src/components/tabBars.js

```diff
@@ -80,24 +80,24 @@ export const TabBar = ({
   return div({
     role: 'navigation',
     'aria-label': props['aria-label'], // duplicate the menu's label on the navigation element
-    'aria-labelledby': props['aria-labelledby']
+    'aria-labelledby': props['aria-labelledby'],
+    style: Style.tabBar.container
   }, [
     h(HorizontalNavigation, {
       role: 'menu',
       'aria-orientation': 'horizontal',
-      style: Style.tabBar.container,
+      style: { display: 'flex', flexGrow: 1, height: '100%' },
       ...props
     }, [
-      ..._.map(([i, name]) => navTab(i, name), Utils.toIndexPairs(tabNames)),
-      div({ style: { flexGrow: 1 } }),
-      children
-    ])
+      ..._.map(([i, name]) => navTab(i, name), Utils.toIndexPairs(tabNames))
+    ]),
+    div({ style: { display: 'flex', flexGrow: 0, alignItems: 'center' } }, children)
   ])
 }
 TabBar.propTypes = {
   activeTab: PropTypes.string,
   tabNames: PropTypes.arrayOf(PropTypes.string).isRequired,
-  displayNames: PropTypes.arrayOf(PropTypes.string),
+  displayNames: PropTypes.object,
```

Comment:
Fixes an error I noted in the JavaScript console, as [WorkflowDetails](https://github.com/DataBiosphere/terra-ui/blob/3338c6dd0c26b77a7e05dd37b52c0fecd86bc069/src/pages/workflows/workflow/WorkflowDetails.js#L101) actually uses the displayNames object-- 
![image](https://user-images.githubusercontent.com/484484/232132841-ecf45f27-b968-40bd-828c-7829f48fee23.png)